### PR TITLE
feat(Apple Music): Artwork as largeImageKey

### PR DESCRIPTION
**Pull request by down-bad**
image

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
bump schema to 1.6

**Review by Bas950 — DISMISSED** [review]

**Comment by EncryptedDev** [comment]
This PR will be merged one #4946 has been merged

**Review by EncryptedDev — DISMISSED** [review]

**Comment by Bas950** [comment]
Update your fork, and fix any merge conflicts it may have.

**Comment by down-bad** [comment]
uh oh

**Review by EncryptedDev — DISMISSED** [review]
lgtm (good song too in proof)

File: websites/T/Twitch/dist/metadata.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "devstatus.twitch.tv",
     "status.twitch.tv"
   ],
-  "version": "3.1.8",
+  "version": "3.1.9",
   "logo": "https://i.imgur.com/So7h3G0.png",
```
Comment (Bas950) [inline websites/T/Twitch/dist/metadata.json:44]: ??

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/T/Twitch/dist/metadata.json (hunk 1 above)
Comment (down-bad) [reply websites/T/Twitch/dist/metadata.json:44]: accidentally:/

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**